less/home-page.less

```
@@ -2,33 +2,102 @@
 
 .home {
 
+  @media @phone {
+    .custom-layout(center, column);
+    border: 6px solid black;
+  }
+
+
+.intro-container {
+
+  @media @phone {
+    .custom-layout(center, column);
+    // border: 5px solid red;
+
+  }
+   
+  .earth {
+
+    @media @phone {
+      height: 11rem;
+  
+    }
+
+    
+  }
+
   .intro {
 
+    
+    @media @phone {
+      .custom-layout(center, column);
+      // border: 5px dotted red;
+      height: 20rem;
+  
+    }
+
     h2 {
       padding: 15px;
+      margin-left: 0px;
     }
 
     p {
       padding:0 15px;
     }
   
-  }
+  } // intro
 
   .space-images {
     display: flex;
+    justify-content: space-between;
     flex-wrap: wrap;
     width: 100%;
```

This takes care of those images

```suggestion
    margin: 5px;
    width: 95%;
    @media @phone {
      // flex-wrap: nowrap;
      // border: 5px solid gray;
      .blast {
        width: 95%;
      };
      .sky {
      display: none;
      };
      .launch {
        display: none;
      }
```

---

This review thing is awkward, but basically you can replace everything inside .space-images starting at line 54 with this. This will get those two images to disappear and the other one to blow up.